gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineFormat.java

```diff
@@ -67,15 +63,24 @@ public void apply(Project project) {
                 if (eclipseFormattingEnabled(project) && palantirJavaFormatterEnabled(project)) {
                     throw new GradleException(
                             "Can't use both eclipse and palantir-java-format at the same time, please delete one of "
-                                    + ECLIPSE_FORMATTING + " or " + PJF_PROPERTY + " from your gradle.properties");
+                                    + ECLIPSE_FORMATTING
+                                    + " or "
+                                    + PJF_PROPERTY
+                                    + " from your gradle.properties");
```

Comment:
I reformatted the whole file using the formatter so a few other things have changed, sorry!